users/users-router.js

```diff
@@ -0,0 +1,16 @@
+const router = require('express').Router();
+
+const Users = require('./users-model.js');
+const restricted = require('../auth/restricted-middleware.js');
+
+router.get('/', restricted, (req, res) => {
+  const department = req.decodedJwt;
+  console.log(department);
+  Users.findByDepartment(department)
+  .then(users => {
+    res.status(201).json(users);
+  })
+  .catch(err => res.send(err));
```

Comment:
When you send data back without a `status` method, the default is HTTP code 200 (OK)! This should use a 500 (Server Error):
```javascript
.catch(err => res.status(500).send(err));
```